elastic/kibana
# [Actionable Observability] Refactor alert search bar

Closes #142713

## Summary
The following changes were applied to the search bar as a prerequisite to sharing this component via the triggers_action_ui plugin:
1. Hide filters since they are not working currently and will be added in a separate ticket (https://github.com/elastic/kibana/issues/140573)
2. Remove refreshNow logic and use absolute time in creating es query (discover also uses absolute time)
3. Use already wired-up `SearchBar` provided by unified search (as suggested in https://github.com/elastic/kibana/issues/116135)
4. Save the status filter separate from the query input in the URL for easier separation between shared logic and custom implementation (In the future, we will use controls for extra filtering that will be provided by the unified search team)

## How to test
1. Go to Alerts page in Observability
2. Make sure the alerts are shown correctly
3. For active/recovered filter, time range and query input filter
    - It should filter alerts correctly
    - The filters should be saved in the URL correctly
    - By refreshing the page, the correct filter should be applied based on the URL query parameter
